src/git/devcontainer-feature.json

```
@@ -1,6 +1,6 @@
 {
     "id": "git",
-    "version": "1.1.6",
+    "version": "1.1.7",
```

```suggestion
    "version": "1.2.0",
```

Can we bump the minor version as this is a nice and significant change?

---

Commit with change coming soon.